lambdatimes/src/components/Content/Tabs.js

```diff
@@ -1,16 +1,30 @@
 import React from 'react';
 import Tab from './Tab';
+import PropTypes from 'prop-types';
+
 const Tabs = props => {
+  let tabs = props.tabs.map((tab, index) => {
+    return <Tab key={index} tab={tab} selectedTab={props.selectedTab} selectTabHandler={props.selectTabHandler}/>; 
```

Comment:
This looks like it could be broken up for readability. For example:
```javascript
return (
  <Tab 
    key={index} 
    tab={tab} 
    selectedTab={props.selectedTab} 
    selectTabHandler={props.selectTabHandler}
  />
); 
```